arkouda/util.py

```diff
@@ -34,7 +34,7 @@ def get_callback(x):
 def concatenate(items, ordered=True):
     warn(
         "This function is deprecated and will be removed in a later version of Arkouda."
-        " Use arkouda.util.generic_concat(items, ordered) instead.",
+        " Use arkouda.server_util.generic_concat(items, ordered) instead.",
```

Comment:
This is referring to the `generic_concat` function in `arkouda/util.py`.  Since nothing in the`arkouda` directory was renamed, this should not be changed.

Reply:
Fixed